ex1/main.css

```diff
@@ -13,5 +13,17 @@
 .center .button {
   height: 50px;
   width: 80px;
-  background-color: #FF6633;
+}
+
+ img {
+  display: block;
+  display: none;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.newButton .centerOfScreen {
```

Comment:
css class names are written like this: new-button and not camelCase.  Please change all of them to lower case letters with hypens between 2 or more words